service/pom.xml

```diff
@@ -32,8 +32,8 @@ under the License.
     <name>Apache Griffin :: Web Service</name>
 
     <properties>
-        <hadoop.version>2.7.1</hadoop.version>
-        <hive.version>2.2.0</hive.version>
+        <hadoop.version>3.0.0-cdh6.3.2</hadoop.version>
+        <hive.version>2.1.1-cdh6.3.2</hive.version>
```

Comment:
I think apache version is better?  @guoyuepeng @chitralverma  WDYT

Reply:
I agree